lib/ImportFromC/HeaderToModel.cpp

```diff
@@ -587,15 +587,13 @@ bool DeclVisitor::VisitFunctionDecl(const clang::FunctionDecl *FD) {
                                                     StackAnnotation,
                                                     Errors);
         if (Stack.has_value()) {
-          // TODO: Handle stack location.
-          //
-          // TODO: Don't forget to assert that it is always a pointer.
-          Errors.emplace_back("revng: We don't support stack return values in "
-                              "RFTs for now.\n");
+          Errors.emplace_back("revng: RFTs must not have stack return values, "
+                              "use SPTAR instead.\n");
```

Comment:
This is my way of contrasting `clang:` prefix. It would be weird for an imaginary error message list to looks something like
```
clang: blah-blah-from-clang
blah-blah-model-verification-failed
clang: not-sure-if-this-can-happen-but-who-knows-how-code-evolves
```

That being said, I'm not too attached, dropping the suffix is a mater of running a regex.

Reply:
I'd simply say "Cannot attach `_STACK` to return values in a RawFunctionDefinition". Replace `_STACK` with whatever annotation we're using.

In order to comment on the error messages, I'd need to see what's the current output.

Reply:
1. 👌.
2. That's a bit tricky, because for now the error reporting is just a vector of strings that eventually gets concatenated. Some of those strings come from clang (not sure if they can contain warnings (I didn't touch that part of the codebase that much, just turned it from "keep only the last error" to "just add every new error to the vector"), but I'd assume so), and some - from us (mostly when annotation don't make sense, or when the model stops verifying after the edit). I feel like there's a value in differentiating where the message is coming from, but not sure how much attention we should put into that for now. Once we have a proper error handling support, this whole thing will have to get redesigned anyway.

Reply:
Done.

P. S. As for the current output samples, search for `expected-error.txt` files in the repo.